Src/PRuntimes/PJavaRuntime/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Do we need this license file? Can we just inherit the P license?

Reply:
Good point, very likely we can!